initialise/initialise.go

```diff
@@ -35,21 +35,28 @@ const (
 
 var kafkaProducerNames = []string{"Observation", "ErrorReporter"}
 
+var kafkaOffset = kafka.OffsetOldest
```

Comment:
I think we would like to be able to change the offset policy from config. Can you add the offset as a config parameter, please? I don't know if you did this for other services, I would suggest something like a boolean config:
```
cfg.KafkaOffsetOldest
```
with a default value of true. Then when you set the offset value depending on that:
```
if cfg.KafkaOffsetOldest{
kafkaOffset := kafka.OffsetOldest
} else {
kafkaOffset := kafka.OffsetNewest
}
```